src/assets/icons/frenchpress.svg

```diff
@@ -0,0 +1,18 @@
+<path d="M174.906 182.331C173.683 180.491 171.2 179.991 169.36 181.214C167.521 182.437 167.021 184.92 168.243 186.76L182.824 208.696C184.047 210.536 186.53 211.036 188.37 209.813C190.209 208.59 190.709 206.108 189.487 204.268L174.906 182.331Z" fill="black"/>
+<path d="M166.364 69C169.125 69 171.364 71.2386 171.364 74C171.364 76.7614 169.125 79 166.364 79L66.3636 79C63.6022 79 61.3636 76.7614 61.3636 74C61.3636 71.2386 63.6022 69 66.3636 69L166.364 69Z" fill="black"/>
+<path d="M142.364 41C145.125 41 147.364 43.2386 147.364 46C147.364 48.7614 145.125 51 142.364 51L91.3636 51C88.6022 51 86.3636 48.7614 86.3636 46C86.3636 43.2386 88.6022 41 91.3636 41L142.364 41Z" fill="black"/>
+<path d="M143.864 56C145.797 56 147.364 57.567 147.364 59.5C147.364 61.433 145.797 63 143.864 63L89.8636 63C87.9306 63 86.3636 61.433 86.3636 59.5C86.3636 57.567 87.9306 56 89.8636 56L143.864 56Z" fill="black"/>
+<path d="M182.864 70C184.797 70 186.364 71.567 186.364 73.5C186.364 75.433 184.797 77 182.864 77H154.864C152.931 77 151.364 75.433 151.364 73.5C151.364 71.567 152.931 70 154.864 70H182.864Z" fill="black"/>
+<path d="M88.6051 56.4612C90.2153 55.6174 92.2047 56.2386 93.0485 57.8488C93.8923 59.459 93.2711 61.4484 91.6609 62.2922L67.2268 75.097C65.6166 75.9408 63.6272 75.3195 62.7834 73.7093C61.9396 72.0991 62.5608 70.1098 64.171 69.2659L88.6051 56.4612Z" fill="black"/>
+<path d="M179.243 74.0383C179.942 72.3603 181.869 71.567 183.547 72.2663C185.225 72.9657 186.018 74.8929 185.319 76.5709L174.706 102.034C174.007 103.712 172.08 104.505 170.402 103.806C168.724 103.106 167.931 101.179 168.63 99.5011L179.243 74.0383Z" fill="black"/>
+<path d="M146.105 56.6246C144.393 55.7274 142.278 56.388 141.381 58.1001C140.483 59.8122 141.144 61.9276 142.856 62.8248L168.62 76.3262C170.332 77.2234 172.447 76.5628 173.344 74.8507C174.242 73.1385 173.581 71.0232 171.869 70.126L146.105 56.6246Z" fill="black"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M64.4711 157.585C54.6478 162.528 42.679 158.572 37.7363 148.749C37.3125 147.907 36.9489 147.037 36.6484 146.144L27.0376 117.547C25.5108 113.004 25.6735 108.063 27.4959 103.631C29.3184 99.1992 32.6782 95.5731 36.9588 93.4185L45.4904 89.1258C52.6491 85.5238 60.9456 86.647 66.8337 91.3224L68.238 116.348L64.8041 103.881C64.4197 102.486 63.736 101.192 62.8006 100.088C61.8651 98.9849 60.7004 98.0985 59.3876 97.491C58.0747 96.8834 56.6451 96.5693 55.1985 96.5705C53.7519 96.5717 52.3228 96.8882 51.011 97.4979L50.7319 97.6327L42.1914 101.93C40.1849 102.939 38.5788 104.598 37.634 106.636C36.6893 108.673 36.4614 110.971 36.9875 113.155L37.0678 113.467L45.1557 142.83C45.5469 144.25 46.2484 145.566 47.2097 146.682C48.1709 147.799 49.3679 148.688 50.7144 149.285C52.0608 149.883 53.5231 150.174 54.9958 150.139C56.4685 150.103 57.9149 149.74 59.2307 149.078L67.7712 144.781C68.4981 144.415 69.1637 143.969 69.7612 143.459L70.386 154.609L64.4711 157.585Z" fill="black"/>
+</svg>
```

Comment:
I did a rebase, and I think in the process this got co-opted into this PR. I am researching how to fix this, but more advanced Git-fu is unfortunately where I have a weak spot. Do you have any suggestions?